Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -140,10 +140,10 @@ public List<Result> ContextMenus(IPublicAPI api)
                     Title = api.GetTranslation("flowlauncher_plugin_program_open_containing_folder"),
                     Action = _ =>
                     {
-
-
-                        Main.StartProcess(Process.Start, new ProcessStartInfo("explorer", ParentDirectory));
-
+                        Main.StartProcess(Process.Start, new ProcessStartInfo(
+                            !string.IsNullOrWhiteSpace(Main._settings.CustomizedExplorer) ? Main._settings.CustomizedExplorer:Settings.Explorer,
+                            !string.IsNullOrWhiteSpace(Main._settings.CustomizedArgs)?Main._settings.CustomizedArgs.Replace("%s",$"\"{ParentDirectory}\"").Replace("%f",$"\"{FullPath}\""):
+                            Settings.ExplorerArgs));
```

Comment:
For readability, what do you think if we do the if statement layout like this:
![image](https://user-images.githubusercontent.com/26427004/98522121-69dfa480-22c8-11eb-9083-d1b6c43f533f.png)

If you like it I can push this change up

Reply:
Sure